src/commands/synthetics/renderer.ts

```diff
@@ -230,18 +230,22 @@ export const renderResults = (test: Test, results: PollResult[], baseUrl: string
 }
 
 // Other rendering
-export const renderTrigger = (test: Test | undefined, testId: string, config: ConfigOverride) => {
+export const renderTrigger = (test: Test, testId: string, executionRule: ExecutionRule, config: ConfigOverride) => {
   const idDisplay = `[${chalk.bold.dim(testId)}]`
 
   const getMessage = () => {
-    if (!test) {
-      return chalk.red.bold(`Could not find test "${testId}"`)
```

Comment:
Should already be handled here: https://github.com/DataDog/datadog-ci/blob/19836a630f094797880f6df367b119fc0125ec03/src/commands/synthetics/utils.ts#L272-L279